solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -638,6 +638,47 @@ describe("RandomBeacon - Relay", () => {
         }
       )
 
+      context(
+        "when timeout happens after all the group terminate or expire",
```

Comment:
Isn't it covered by the next context: when no active groups exist after timeout is reported? Line 682.

Reply:
I wanted to cover the scenario described in [this comment](https://github.com/keep-network/keep-core/pull/2686#discussion_r750128611). Reworded this context, if you have a better idea, let's do it. 07d528f